precompile-patches/pico.toml

```diff
@@ -1,16 +1,7 @@
 [patch.crates-io]
```

Comment:
This was quite messy to figure out. I removed everything that generated a warning, always trying to double check if there's a correct patch to be applied.

I also asked Brevis people if they've a place to double check what they expect should apply. I haven't got an answer yet, but if they provide something to double check and there's something missing, I'll add in further PR.